tensorhive/api/APIServer.py

```diff
@@ -2,29 +2,26 @@
 import connexion
 from tensorhive.config import API_CONFIG
 from tensorhive.database import db_session
+log = logging.getLogger(__name__)
 
 
 class APIServer():
     def start(self):
-        # TODO Enable debug mode
-        # TODO Read settings from config
-        # FIXME: set log level for the whole app in cli
-        logging.basicConfig(level=logging.INFO)
         app = connexion.FlaskApp(__name__)
+        log.info(
+            'API docs (Swagger UI) available at: http://<address>:<port>/v1.0/ui/')
```

Comment:
Is it intentional, that address and port are not given explicitly? I would expect a working URL.

Reply:
Fixed: 5b907841ad4fb6cf03bfa7acdb7ad0bfa126a0e1